### PR TITLE
Explain how to use Dask to parallelize grid search across GPUs

### PR DESCRIPTION
**Pull request by dnouri**
Based on #237 

Work in progress, but please give me feedback.

Also explains how to use Palladium to run grid search in parallel.  This feature currently requires https://github.com/ottogroup/palladium/pull/87

### CODE REVIEW

File: docs/user/parallelism.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Next, assuming you have two GPUs on your machine, let's start up a
+Dask scheduler and two Dask workers::
+
+  dask-scheduler
+  CUDA_VISIBLE_DEVICES=0 dask-worker 127.0.0.1:8786 --nthreads 1
+  CUDA_VISIBLE_DEVICES=1 dask-worker 127.0.0.1:8786 --nthreads 1
+
+Now, instead of importing ``GridSearchCV`` or ``RandomizedSearchCV``
+from scikit-learn, import from ``dask_searchcv`` instead.  And set up
```
Comment (benjamin-work) [inline docs/user/parallelism.rst]: "instead ... instead." Remove the 2nd "instead"

**Comment by benjamin-work** [comment]
addresses #95

**Comment by dnouri** [comment]
Updated to use joblib's `parallel_backend` instead of `dask-searchcv`.

**Comment by benjamin-work** [comment]
Thanks, great work.

**Pull request merged**